ocfweb/announcements/announcements.py

```diff
@@ -123,3 +127,18 @@ def hpc_survey(title, request):
             'title': title,
         },
     )
+
+
+@announcement(
+    'The OCF is hiring!',
+    date(2017, 3, 20),
+    'hiring',
+)
+def hpc_survey(title, request):
```

Comment:
Haha, that's what I get for copying and pasting... How did this even work in development?